numpy/core/records.py

```diff
@@ -657,20 +708,27 @@ def fromarrays(arrayList, dtype=None, shape=None, formats=None,
 
 def fromrecords(recList, dtype=None, shape=None, formats=None, names=None,
                 titles=None, aligned=False, byteorder=None):
-    """ create a recarray from a list of records in text form
+    """Create a recarray from a list of records in text form.
 
-        The data in the same field can be heterogeneous, they will be promoted
-        to the highest data type.  This method is intended for creating
-        smaller record arrays.  If used to create large array without formats
-        defined
-
-        r=fromrecords([(2,3.,'abc')]*100000)
-
-        it can be slow.
+    Parameters
+    ----------
+    recList : sequence
+        data in the same field may be heterogeneous - they will be promoted
+        to the highest data type.
+    dtype : data-type, optional
+        valid dtype for all arrays
+    shape : int or tuple of ints, optional
+        shape of each array.
+    formats, names, titles, aligned, byteorder :
+        If `dtype` is ``None``, these arguments are passed to `numpy.format_parser` to construct a dtype.
+        See that function for detailed documentation
 
-        If formats is None, then this will auto-detect formats. Use list of
-        tuples rather than list of lists for faster processing.
```

Comment:
I think we need to keep this.

Reply:
If there was a slick way to use the git or github interface to assist in making this edit, I'd like to know it.

Reply:
yes, you can suggest a change in the ui and the maintainer can accept it.  https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request (point 6)

Reply:
Thanks for the info! Yes, that's what I was missing.